test/solidity/Facets/SymbiosisFacet.t.sol

```diff
@@ -0,0 +1,176 @@
+
+        symbiosisData = SymbiosisFacet.SymbiosisData(
+            "",
+            "",
+            address(0),
+            ADDRESS_USDC,
+            address(0),
+            address(0),
+            RELAY_RECIPIENT,
+            //swap after bridging
+            _otherSideCalldata
+        );
+    }
+
+    function initiateBridgeTxWithFacet(bool isNative) internal override {
+
+        if (isNative) {
+            symbiosisFacet.startBridgeTokensViaSymbiosis{
+            value: bridgeData.minAmount
+            }(bridgeData, symbiosisData);
+        } else {
+            symbiosisFacet.startBridgeTokensViaSymbiosis(
+                bridgeData, symbiosisData
+            );
+        }
+    }
+
+    function initiateSwapAndBridgeTxWithFacet(
+        bool isNative
+    ) internal override {
+
+        if (isNative) {
+            symbiosisFacet.swapAndStartBridgeTokensViaSymbiosis{
+            value: swapData[0].fromAmount
+            }(bridgeData,swapData, symbiosisData);
+        } else {
+
+            symbiosisFacet.swapAndStartBridgeTokensViaSymbiosis{ value: addToMessageValue }(
+                bridgeData,
+                swapData,
+                symbiosisData
+            );
+        }
+    }
+
+
+    function testBase_CanBridgeNativeTokens() public override {
+//        address[] memory path = new address[](2);
```

Comment:
Why commented out code here?

If your plan is to override the test to make it not applicable then you should provide a reason why the test is not applicable (e.g. "//native token bridging not supported by this bridge")